test/test.googleauth.ts

```diff
@@ -18,7 +18,7 @@ import assert from 'assert';
 import child_process from 'child_process';
 import crypto from 'crypto';
 import * as fs from 'fs';
-import {BASE_PATH, HEADER_NAME, HOST_ADDRESS} from 'gcp-metadata';
+import {BASE_PATH, HEADERS, HOST_ADDRESS} from 'gcp-metadata';
```

Comment:
Are `HEADER_NAME` and `HEADERS` both available on the same version of `gcp-metadata`? (If not, I think you'd need to update that dependency?)

Reply:
They're both available at the same time :)